src/cli/argument-parser.js

```diff
@@ -110,6 +110,7 @@ export default class CLIArgumentParser {
             .option('--ssl', 'specify SSL options to run TestCafe proxy server over the HTTPS protocol')
             .option('--proxy-bypass <rules>', 'specify a comma-separated list of rules that define URLs accessed bypassing the proxy server')
             .option('--disable-page-reloads', 'disable page reloads between tests')
+            .option('--development-mode', 'launch TestCafe with non-minified service scripts')
```

Comment:
I think `--development-mode` doesn't describe the purpose of this mode. This mode should be enabled if something goes wrong with TestCafe and you have to retrieve additional information to help us reproduce the problem.  

So I think it's better to call it `--recovery-mode` and provide something like this as a description:
```
'--recovery-mode', 'launch TestCafe in troubleshooting mode to debug TestCafe internals'
```  

\cc @kirovboris